webapp/channels/src/i18n/en.json

```diff
@@ -2728,7 +2729,7 @@
   "admin.system_properties.confirm.delete.title": "Delete {name} property",
   "admin.system_properties.details.saving_changes": "Saving configuration…",
   "admin.system_properties.details.saving_changes_error": "There was an error while saving the configuration",
-  "admin.system_properties.user_properties.add_property": "Add property",
+  "admin.system_properties.user_properties.add_property": "Add attribute",
   "admin.system_properties.user_properties.dotmenu.ad_ldap.link_property.label": "Link property to AD/LDAP",
```

Comment:
"admin.system_properties.user_properties.dotmenu.ad_ldap.link_property.label": "Link attribute to AD/LDAP",